go.work

```diff
@@ -1,7 +1,11 @@
-go 1.22
+go 1.22.0
+
+toolchain go1.23.2
```

Comment:
I think this line is causing problems with the linter.

Reply:
fixed by upgrading all go versions to 1.23